asm/unk_0202EB30.s

```diff
@@ -178,8 +178,8 @@ _0202EC68: .word 0x000005D8
 _0202EC6C: .word 0x00000678
 	thumb_func_end sub_0202EC28
 
-	thumb_func_start sub_0202EC70
-sub_0202EC70: ; 0x0202EC70
+	thumb_func_start GetMigratedPokemonI
```

Comment:
Why MigratedPokemonI?

Reply:
Migrated because it was brought over from gen 3, I because the second param indicates the index of the mograted pokemon (out of 6), and Pokemon because, well, obvious reasons.

Reply:
GetMigratedPokemonIndex

Reply:
Disagree. This function takes an index and populates a Pokemon in the buffer pointed to by the third argument. `GetMigratedPokemonIndex` incorrectly implies you are supplying a Pokemon and expecting an index.